swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0):
+        super().__init__(category, condition)
```

Comment:
Great use of `super` here! Consider removing the `category` from the constructor arguments on line 3. We know we always want it to be "Clothing" so we don't want to allow changing it when instantiating an instance.